src/blocks/Header/Header.tsx

```diff
@@ -121,21 +121,21 @@ export const HeaderBlock = (props: WithChildren<HeaderBlockFullProps>) => {
                             >
                                 <Col sizes={titleSizes} className={b('content-inner')}>
                                     {overtitle && (
-                                        <p className={b('overtitle')}>
```

Comment:
Because the <p> tag in HTML cannot contain block-level elements, like <div>, according to the HTML5 content model. When an HTML parser encounters this scenario, it implicitly closes the <p> tag before the <div> starts and re-opens a new one after the <div> ends, resulting in the observed behavior.

In simple words: browsers automatically closed the p tag when they met div inside, cause they considered div not as a content of p, but rather as the next node after the p.

The description node looked like this:

<p class="pc-header-block__description">
    <div class="yfm">
        <!-- Typographed content started, this is not a part of any component -->
        <p>Text...</p>
        <!-- End of the content -->
    </div>
</p>
In fact, browser renders it this way:

<p class="pc-header-block__description">
    </p><div class="yfm">
        <!-- Typographed content started, this is not a part of any component -->
        <p>Text...</p>
        <!-- End of the content -->
    </div>
<p></p>